sdk-extensions/autoconfigure/README.md

```diff
@@ -0,0 +1,161 @@
+# OpenTelemetry SDK Autoconfigure
+
+This artifact implements environment-based autoconfiguration of the OpenTelemetry SDK. This can be
+an alternative to programmatic configuration using the normal SDK builders.
+
+All options support being passed as Java system properties, e.g., `-Dotel.traces.exporter=zipkin` or
+environment variables, e.g., `OTEL_TRACES_EXPORTER=zipkin`.
+
+## Contents
+
+* [Exporters](#exporters)
+  + [OTLP exporter (both span and metric exporters)](#otlp-exporter--both-span-and-metric-exporters-)
+  + [Jaeger exporter](#jaeger-exporter)
+  + [Zipkin exporter](#zipkin-exporter)
+  + [Prometheus exporter](#prometheus-exporter)
+  + [Logging exporter](#logging-exporter)
+* [Trace context propagation](#propagator)
+* [OpenTelemetry Resource](#opentelemetry-resource)
+* [Batch span processor](#batch-span-processor)
+* [Sampler](#sampler)
+* [Span limits](#span-limits)
+* [Interval metric reader](#interval-metric-reader)
+* [Customizing the OpenTelemetry SDK](#customizing-the-opentelemetry-sdk)
+
+## Exporters
```

Comment:
Should we put links to spec to each section?

Reply:
Could, but I think for end users reading this the SDK spec will generally not be interesting since that's handled by us. Semantic convention spec could be I guess, I did add one link for `service.name`.